Content/ContentTypeResolver/SnippetSelectionResolver.php

```diff
@@ -88,27 +86,13 @@ public function resolve($data, PropertyInterface $property, string $locale, arra
             }
 
             $snippet->setIsShadow(null !== $shadowLocale);
-            $snippet->setShadowBaseLanguage($shadowLocale);
+            /** @var string $shadowBaseLanguage */
+            $shadowBaseLanguage = $shadowLocale;
+            $snippet->setShadowBaseLanguage($shadowBaseLanguage);
 
-            $snippets[] = $this->structureResolver->resolve($snippet, $locale, $loadExcerpt);
+            $snippets[] = $this->structureResolver->resolve($snippet, $locale, $includeExtension);
         }
 
-        return new ContentView($snippets, ['ids' => $data ?: []]);
-    }
-
-    private function getDefaultSnippetId(string $webspaceKey, string $snippetArea, string $locale): ?string
-    {
-        try {
-            /** @var SnippetDocument|null $snippet */
-            $snippet = $this->defaultSnippetManager->load($webspaceKey, $snippetArea, $locale);
-        } catch (WrongSnippetTypeException $exception) {
-            return null;
-        }
-
-        if (!$snippet) {
-            return null;
-        }
-
-        return $snippet->getUuid();
+        return new ContentView($snippets, ['ids' => $snippetIds ?: []]);
```

Comment:
not sure if we returned `$data` (does not contain default snippet id) instead of `$snippetIds` (does contain default snippet id) here intentionally. cant think how someone would use this so its okay for me both ways 🙂

Reply:
I just thought, it would be confusing, if an actual snippet is returned (the default snippet) but the view data says the id is null